### PR TITLE
Enable service_account_creds test and compute_engine_creds for gRpc-java.

### PR DESCRIPTION
**Pull request by madongfly**
@ejona86 @donnadionne

### CODE REVIEW

**Comment by madongfly** [comment]
@donnadionne, is there any script that I can run to  verify the change is effective?

File: tools/gce_setup/grpc_docker.sh [hunk 1]
```diff
@@ -1274,8 +1274,7 @@ grpc_cloud_prod_gen_go_cmd() {
 #   cmd=$($grpc_gen_test_cmd $flags)
 grpc_interop_gen_java_cmd() {
     local cmd_prefix="sudo docker run grpc/java";
-    local test_script="/var/local/git/grpc-java/run-test-client.sh";
-    local test_script+=" --use_test_ca=true --use_tls=true"
+    local test_script="/var/local/git/grpc-java/run-test-client.sh --use_test_ca=true --use_tls=true";
```
Comment (ejona86) [inline tools/gce_setup/grpc_docker.sh]: Why this change and the similar one below?

Comment (madongfly) [inline tools/gce_setup/grpc_docker.sh]: Just cleanup, having two lines is unnecessary and different than all the other functions.

Comment (ejona86) [inline tools/gce_setup/grpc_docker.sh]: I think it was the equivalent of word wrapping. grpc_cloud_prod_gen_go_cmd does the same thing. grpc_cloud_prod_auth_compute_engine_creds_gen_ruby_cmd too. grpc_interop_gen_php_cmd too. Which ones are you seeing that don't do it?

**Comment by ejona86** [comment]
This looks fine to me. It does seem @donnadionne would be best to review though.

File: tools/gce_setup/grpc_docker.sh (hunk 1 above)
Comment (madongfly) [inline tools/gce_setup/grpc_docker.sh]: I see, the bunch of "node" and "csharp" tests just below the java tests misled me.

Let me know if you want they reverted, otherwise I won't bother to change it back.
